addons/hr_expense/models/hr_expense.py

```diff
@@ -601,6 +602,7 @@ def action_move_create(self):
                 'line_ids':[
                     Command.create({
                         'name': expense.employee_id.name + ': ' + expense.name.split('\n')[0][:64],
+                        'account_id': expense.account_id.id,
```

Comment:
As the field is not required, are we sure it will be set?  Also, normally this code is put in a _prepare_ statement so we can inherit more easily (but it is outside of our scope).

Reply:
@jco-odoo I tested this, and if the field is not set, it will show a user error when trying to post the entries. It seems to be required in the expense for the account move to be created.